assignments/callbacks.js

```diff
@@ -19,29 +19,66 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
     console.log(first)
   });
 
-*/
 
 
-function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
-}
 
-function last(arr, cb) {
-  // last passes the last item of the array into the callback.
-}
+  function getLength(arr, cb) {
+    // getLength passes the length of the array into the callback.
+    return cb(items.length)
+  }
 
-function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
-}
+  getLength(items, function(length) {
+    console.log(length)
+  });
 
-function multiplyNums(x, y, cb) {
-  // multiplyNums multiplies two numbers and passes the result to the callback.
-}
 
-function contains(item, list, cb) {
-  // contains checks if an item is present inside of the given array/list.
-  // Pass true to the callback if it is, otherwise pass false.
-}
+
+
+  function last(arr, cb) {
+    // last passes the last item of the array into the callback.
+    return cb(arr[3])
+  }
+
+  last(items, function(last) {
+    console.log(last)
+  });
+
+
+
+  function sumNums(x, y, cb) {
+    // sumNums adds two numbers (x, y) and passes the result to the callback.
+    return cb (x + y)
+  }
+
+  sumNums(3, 2, function(add) {
+    console.log(add)
+  })
+
+
+
+  function multiplyNums(x, y, cb) {
+    // multiplyNums multiplies two numbers and passes the result to the callback.
+    return cb (x * y)
+  }
+
+  multiplyNums(2, 9, function(multiply) {
+    console.log(multiply)
+  })
+
+
+
+  function contains(item, list, cb) {
+    // contains checks if an item is present inside of the given array/list.
+    // Pass true to the callback if it is, otherwise pass false.
+    return cb (items.includes(newItem))
```

Comment:
good job, I personally try to avoid using .includes as it can be unreliable at times, in this case you could have also gone with a true or false if statement.
`if (list.item) return true
return false`

^placing the list.item in the statement already makes it a true value if it is there, and make it return true. You also don't really need the brackets in a simple one for one evaluation. You could also write this with a ternary statement such as
`item.list ? true : false`